public/app/features/plugins/sql/components/visual-query-builder/SQLWhereRow.tsx

```diff
@@ -49,3 +53,14 @@ function mapFieldsToTypes(columns: SQLSelectableValue[]) {
   }
   return fields;
 }
+
+export function removeQuotesForMultiVariables(val: SQLExpression, templateVars: VariableWithMultiSupport[]) {
+  if (
+    templateVars.some((tv) => {
```

Comment:
Since the return value from this arrow function is acting as a condition for the `if` statement, I would consider either moving the logic it to it's own function which you could call here - for readability - or defining a variable above the if statement encompassing said logic. I find when working with conditional statements, the less code in the condition, the easier it is to reason.

Reply:
+1 on this

Reply:
Makes sense to me! I'll push through that change quickly now.